apps/experiments/models.py

```diff
@@ -747,6 +747,13 @@ def compare_with_latest(self):
             version.compare(prev_version.version)
         return version.fields_changed
 
+    def archive_working_experiment(self):
```

Comment:
This looks like it can be done inside the existing `archive` method?

Reply:
I think the logic here was that if versions could have channels then you'd want to call `.archive` on the versions to also archive the channels. 

But since you can't attach channels to a version I don't think it's necessary to do it that way.